lib/error-matcher.js

```diff
@@ -78,22 +91,23 @@ module.exports = (function () {
   };
 
   ErrorMatcher.prototype.set = function (regex, cwd, output) {
-    try {
-      this.regex = XRegExp(regex);
-    } catch (err) {
-      this.regex = null;
-      return this.emit('error', 'Error parsing regex:\n' + err);
-    }
+    regex = (regex instanceof Array) ? regex : [ regex ];
+
+    this.regex = _.compact(regex.map(function(r) {
+      try {
+        return XRegExp(r);
+      } catch (err) {
+        this.emit('error', 'Error parsing regex:\n' + err);
```

Comment:
this is not bound here. You need to add `.bind(this)` after the last `}` in the function declaration.